packages/babel-cli/src/babel/dir.js

```diff
@@ -135,27 +134,23 @@ export default async function ({
   let compiledFiles = 0;
   let startTime = null;
 
-  const logSuccess = debounce(
-    function () {
-      if (startTime === null) {
-        // This should never happen, but just in case it's better
-        // to ignore the log message rather than making @babel/cli crash.
-        return;
-      }
+  const logSuccess = util.debounce(function () {
```

Comment:
oh great! didn't know we had an util for this, as I wasn't sure about replacing in the earlier PR because of the `trailing: true` and also `logSuccess.flush();` on line 173, but looks like it covers it!